runtime/onert/api/nnfw/include/nnfw_experimental.h

```diff
@@ -233,9 +243,9 @@ typedef struct nnfw_train_info
   /** optimizer type */
   NNFW_TRAIN_OPTIMIZER opt = NNFW_TRAIN_OPTIMIZER_SGD;
 
-  /** indexes of trainable operations */
-  uint32_t trainble_ops_size = 0;
-  uint32_t trainble_ops_idx[NNFW_TRAINABLE_OPS_MAX_SIZE];
+  /** Number of layers to be trained from the back. "-1" means that all layers will be trained.
+   * "0" means that no layer will be trained. Negative value less than -1 means error. */
+  int32_t num_of_trainable_ops = 0;
```

Comment:
```suggestion
  int32_t num_of_trainable_ops = NNFW_TRAIN_TRAINABLE_NONE;
```

I like this approach. Thank you. :)

Reply:
added ;)